<ui version="4.0" >
 <class>QAjOptionsDialogBase</class>
 <widget class="QDialog" name="QAjOptionsDialogBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Configure</string>
  </property>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>451</y>
     <width>381</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>131</width>
        <height>31</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="okButton" >
      <property name="text" >
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton" >
      <property name="text" >
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTabWidget" name="tabWidget" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>20</y>
     <width>381</width>
     <height>421</height>
    </rect>
   </property>
   <property name="currentIndex" >
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab" >
    <attribute name="title" >
     <string>Common</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget" >
     <property name="geometry" >
      <rect>
       <x>11</x>
       <y>14</y>
       <width>351</width>
       <height>365</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="5" column="2" >
         <widget class="QToolButton" name="incomingButton" >
          <property name="text" >
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="label_3" >
          <property name="text" >
           <string>Retype Password:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2" >
         <widget class="QToolButton" name="tempButton" >
          <property name="text" >
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" >
         <widget class="QLineEdit" name="coreEdit" />
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>New Password:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0" >
         <widget class="QLabel" name="label_8" >
          <property name="text" >
           <string>Server Source:</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0" >
         <widget class="QLabel" name="label_9" >
          <property name="text" >
           <string>Refresh:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1" >
         <widget class="QLineEdit" name="serverEdit" />
        </item>
        <item row="3" column="0" >
         <widget class="QLabel" name="label_4" >
          <property name="text" >
           <string>Core Address:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Nick:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1" >
         <widget class="QLineEdit" name="xmlEdit" />
        </item>
        <item row="5" column="1" >
         <widget class="QLineEdit" name="incomingEdit" />
        </item>
        <item row="6" column="1" >
         <widget class="QLineEdit" name="tempEdit" />
        </item>
        <item row="1" column="1" >
         <widget class="QLineEdit" name="passwordEdit" />
        </item>
        <item row="5" column="0" >
         <widget class="QLabel" name="label_6" >
          <property name="text" >
           <string>Incoming Directory:</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1" >
         <widget class="QSpinBox" name="refreshSpin" >
          <property name="maximumSize" >
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="suffix" >
           <string>sec.</string>
          </property>
          <property name="minimum" >
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="6" column="0" >
         <widget class="QLabel" name="label_7" >
          <property name="text" >
           <string>Temporary Directory:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QLineEdit" name="retypeEdit" />
        </item>
        <item row="4" column="0" >
         <widget class="QLabel" name="label_5" >
          <property name="text" >
           <string>XML Port:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLineEdit" name="nickEdit" />
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QCheckBox" name="autoConnect" >
          <property name="text" >
           <string>automatically connect to servers</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="savePassword" >
          <property name="text" >
           <string>save core password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="showSplash" >
          <property name="text" >
           <string>show splash screen on startup</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2" >
    <attribute name="title" >
     <string>Network</string>
    </attribute>
    <widget class="QWidget" name="widget" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>10</y>
       <width>351</width>
       <height>213</height>
      </rect>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="2" column="1" >
       <widget class="QSpinBox" name="slotSpin" >
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
        <property name="suffix" >
         <string> kByte/s</string>
        </property>
        <property name="maximum" >
         <number>9999</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QSpinBox" name="sourcesSpin" >
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
        <property name="suffix" >
         <string/>
        </property>
        <property name="maximum" >
         <number>9999</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0" >
       <widget class="QLabel" name="label_15" >
        <property name="text" >
         <string>Max. New Connections:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="QSpinBox" name="connectionsSpin" >
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
        <property name="suffix" >
         <string/>
        </property>
        <property name="maximum" >
         <number>9999</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_11" >
        <property name="text" >
         <string>Max. Upload Speed:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_10" >
        <property name="text" >
         <string>Max. Download Speed:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1" >
       <widget class="QLineEdit" name="tcpEdit" />
      </item>
      <item row="1" column="1" >
       <widget class="QSpinBox" name="upSpin" >
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
        <property name="suffix" >
         <string> kByte/s</string>
        </property>
        <property name="maximum" >
         <number>9999</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_12" >
        <property name="text" >
         <string>Max. Speed Per Slot:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" >
       <widget class="QLabel" name="label_16" >
        <property name="text" >
         <string>TCP Port:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" >
       <widget class="QSpinBox" name="newSpin" >
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
        <property name="suffix" >
         <string> per 10 seconds</string>
        </property>
        <property name="maximum" >
         <number>9999</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="label_14" >
        <property name="text" >
         <string>Max. Connections:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_13" >
        <property name="text" >
         <string>Max. Sources Per File:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QSpinBox" name="downSpin" >
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
        <property name="suffix" >
         <string> kByte/s</string>
        </property>
        <property name="maximum" >
         <number>9999</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3" >
    <attribute name="title" >
     <string>Launching</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget" >
     <property name="geometry" >
      <rect>
       <x>12</x>
       <y>13</y>
       <width>351</width>
       <height>261</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label_22" >
          <property name="text" >
           <string>Open files with:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="launchCombo" >
          <property name="editable" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox" >
        <property name="title" >
         <string>Location of files</string>
        </property>
        <widget class="QWidget" name="layoutWidget" >
         <property name="geometry" >
          <rect>
           <x>10</x>
           <y>22</y>
           <width>321</width>
           <height>201</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QRadioButton" name="sameComputerRadio" >
            <property name="text" >
             <string>Core is running at the same computer</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="specificRadio" >
            <property name="text" >
             <string>Use specific folders (e.g. for smb):</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item row="1" column="1" >
             <widget class="QToolButton" name="incomingSpecificButton" >
              <property name="text" >
               <string>...</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" >
             <widget class="QLabel" name="tempSpecificLabel" >
              <property name="text" >
               <string>Temporary Directory:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" >
             <widget class="QLineEdit" name="incomingSpecificEdit" />
            </item>
            <item row="0" column="0" >
             <widget class="QLabel" name="incomingSpecificLabel" >
              <property name="text" >
               <string>Incoming Directory:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0" >
             <widget class="QLineEdit" name="tempSpecificEdit" />
            </item>
            <item row="3" column="1" >
             <widget class="QToolButton" name="tempSpecificButton" >
              <property name="text" >
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QRadioButton" name="ftpRadio" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="text" >
             <string>Use ftp</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="ftp_tab" >
    <attribute name="title" >
     <string>Ftp</string>
    </attribute>
    <widget class="QWidget" name="widget" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>10</y>
       <width>351</width>
       <height>151</height>
      </rect>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="ftpServerEdit" />
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_18" >
        <property name="text" >
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_19" >
        <property name="text" >
         <string>User:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QLineEdit" name="ftpPasswordEdit" >
        <property name="echoMode" >
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_17" >
        <property name="text" >
         <string>Server:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="ftpUserEdit" />
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_20" >
        <property name="text" >
         <string>Password:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="ftpPortEdit" />
      </item>
      <item row="4" column="1" >
       <widget class="QLineEdit" name="ftpDirEdit" />
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="label_21" >
        <property name="text" >
         <string>Directory:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>nickEdit</tabstop>
  <tabstop>passwordEdit</tabstop>
  <tabstop>retypeEdit</tabstop>
  <tabstop>coreEdit</tabstop>
  <tabstop>xmlEdit</tabstop>
  <tabstop>incomingEdit</tabstop>
  <tabstop>incomingButton</tabstop>
  <tabstop>tempEdit</tabstop>
  <tabstop>tempButton</tabstop>
  <tabstop>serverEdit</tabstop>
  <tabstop>refreshSpin</tabstop>
  <tabstop>autoConnect</tabstop>
  <tabstop>savePassword</tabstop>
  <tabstop>downSpin</tabstop>
  <tabstop>upSpin</tabstop>
  <tabstop>slotSpin</tabstop>
  <tabstop>sourcesSpin</tabstop>
  <tabstop>connectionsSpin</tabstop>
  <tabstop>newSpin</tabstop>
  <tabstop>tcpEdit</tabstop>
  <tabstop>ftpServerEdit</tabstop>
  <tabstop>ftpPortEdit</tabstop>
  <tabstop>ftpUserEdit</tabstop>
  <tabstop>ftpPasswordEdit</tabstop>
  <tabstop>ftpDirEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>QAjOptionsDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>QAjOptionsDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
